packages/pretty-format/src/__tests__/DOMElement.test.ts

```diff
@@ -395,17 +395,28 @@ Testing.`;
 
     test('jsdom 9 and 10', () => {
       // Mock element objects to make sure the plugin really matches them.
-      function SVGSVGElement(attributes, ...children) {
-        this.nodeType = 1;
-        this.tagName = 'svg'; // lower case
-        this.attributes = attributes;
-        this.childNodes = children;
+      class SVGSVGElement {
```

Comment:
Is it fine to refactor these into classes? The classes are somewhat naive, but probably fine as test fixtures. Another question – is it still relevant to support `jsdom` v9 or v10? Does not hurt, of course (;